crm_integration_xblock/crm_integration_xblock.py

```diff
@@ -34,6 +34,10 @@ class CrmIntegration(StudioEditableXBlockMixin, XBlock):
     and holding the private data we don't want to send to the browsers.
     """
 
+    # Show the XBlock when masquerading as a specific user.
```

Comment:
No need for this message if we accept that course_team users will be able (and should be) to change stuff as the given user

Reply:
Done.